testutil/sample/sample_ton.go

```diff
@@ -8,7 +8,6 @@ import (
 	"unsafe"
```

Comment:
I don't think we need `sample_` prefix in the file name as already in sample package

Reply:
got it. I used sample_ for ease of search via `cmd+f` in the editor